otaclient/app/configs.py

```diff
@@ -77,44 +64,86 @@ class BaseConfig:
     PASSWD_FILE = "/etc/passwd"
     GROUP_FILE = "/etc/group"
     FSTAB_FPATH = "/etc/fstab"
+    # where the OTA image meta store for this slot
+    META_FOLDER = "/opt/ota/image-meta"
```

Comment:
folder and directory are the same meaning and I'd like to unify this terminology to `directory`, in the future. Make sure not this PR is OK.